K_p: [1.50,0,0,0,0,0,
      0,1.50,0,0,0,0,
      0,0,1.50,0,0,0,
      0,0,0,1.50,0,0,
      0,0,0,0,1.50,0,
      0,0,0,0,0,1.50]
K_i: [0.0001,0,0,0,0,0,
      0,0.0001,0,0,0,0,
      0,0,0.0001,0,0,0,
      0,0,0,0.0001,0,0,
      0,0,0,0,0.0001,0,
      0,0,0,0,0,0.001]
K_d: [0.1,0,0,0,0,0,
      0,0.1,0,0,0,0,
      0,0,0.1,0,0,0,
      0,0,0,0.1,0,0,
      0,0,0,0,0.1,0,
      0,0,0,0,0,0.1]


workspace_limits: [-1.0, 1.0, -1.0, 1.0, -0.5, 1.0]

arm_max_vel: 5.0
arm_acc_upper_limit: 0.75
arm_acc_lower_limit: -0.75
frequency: 1000
